### ISSUE TITLE
this is stupid!

### ISSUE THREAD

**Issue by Galaxy-Coding** [madrobby/vapor.js#110]
I demand you to delete this repo to free up space on Github‘s servers so more useful projects with less stars can thrive. This is BS honestly. Nobody is laughing.

**Comment by TheTechRobo** [comment]
@Galaxy-Coding is this a joke?

**Comment by Galaxy-Coding** [comment]
@TheTechRobo not a joke, especially on the “free space on server“ part. Join the petition

**Comment by madrobby** [comment]
The library is already taking up no space ¯\_(ツ)_/¯

**Comment by Galaxy-Coding** [comment]
@madrobby yes but many other files on the repo, the comments, issues, pull requests, config files, readme files, etc.

**Comment by madrobby** [comment]
Especially issues and pull requests!

**Comment by Galaxy-Coding** [comment]
@madrobby sorry if I offended you or sounded rude. Hope you take this seriously

**Comment by TheTechRobo** [comment]
@Galaxy-Coding What I fail to understand is how Microsoft needs 5MB more server space, and why that would help projects with lower stars.

**Comment by duongdominhchau** [comment]
If this is serious, it is not just sounded rude, it is really rude. You came out of nowhere and **demand** the author to delete this repo. If you own GitHub, change the ToS to your wish, why write here? And technically, GitHub does not work the way you write, deleting a repo gives other repos nothing, absolutely nothing.

**Comment by TheTechRobo** [comment]
They might be meaning in the "trending" or "explore repositories" sections, but it's still really rude, and completely unnecessary. This is like when [people demanded removal of 418 http status code from big repos](https://github.com/psf/requests/issues/4238).

**Comment by pepefeliblu** [comment]
I don't understand why you want to delete this as this repo is pretty useful! (Pun intended)

**Comment by TheTechRobo** [comment]
~~where's the pun~~

**Comment by fightborn** [comment]
> I demand you to delete this repo to free up space on Github‘s servers so more useful projects with less stars can thrive. This is BS honestly. Nobody is laughing.

I demand you delete this issue, it's taking up space and GitHub's servers should be used to host useful, constructive issues. Not to cater to whims of people devoid of humor.

Thank you. 

Btw. if you don't delete your issue, I will be extremely offended and you don't want that.

**Comment by tj-commits** [comment]
Don't delete this repo. it's very important.
